api/ingest-tokens.go

```diff
@@ -76,25 +76,30 @@ func toIngestToken(data ingestTokenData) *IngestToken {
 	}
 }
 
-func (i *IngestTokens) Add(repositoryName string, tokenName string, parser string) (*IngestToken, error) {
+func (i *IngestTokens) Add(repositoryName string, tokenName string, parser string, customToken string) (*IngestToken, error) {
```

Comment:
This would be a breaking change for Go API users.

Could we rename it to `AddV2` and call it from `ingest_tokens_add.go` and `Add()` to keep the signature? @SaaldjorMike 

```
func (i *IngestTokens) Add(repositoryName string, tokenName string, parser string) (*IngestToken, error) {
	return i.AddV2(repositoryName, tokenName, parser, "")
}
```